lib/config/load.js

```diff
@@ -30,7 +30,10 @@ function load(settings, options, config, callback) {
       // Then merge over with the user settings (parsed from the cli).
       // Note that merge protects and favours existing values over new values,
       // and thus command line arguments get priority
+      
+      let script = options.script;
       options = utils.merge(settings, options);
+      if (!options.script) options.script = script;
```

Comment:
I appreciate this a very old PR, but I'm not sure how this isn't happening already? If `options.script` exists, it'll land in the `options` value.

Reply:
I remember finding this causing an issue where I couldn't get something working because options were getting overwritten somewhere.

Reply:
This is old, it may not apply anymore if someone else fixed it. I'll try and remember exactly what was happening.

Reply:
After trying to find the cause of my issue here again: I remember debugging this and finding that options.script was not defined even though it was in the config. I can't find the project I was working on that had this problem, I think I have a monkeypatchched version of nodemon somewhere though to work around it.

Reply:
maybe settings has a script property that evaluates to false?